documentation/developers/cli-client.md

```diff
@@ -0,0 +1,19 @@
+# CLI Client
+
+The CLI (command line interface) client can be used to send RPC commands from command line to Phoniebox.
+
+## Installation
+
+* Install prerequisites: `sudo apt-get install libczmq-dev`
+* Change to directory: `cd ~/RPi-Jukebox-RFID/src/cli_client`
+* Compile CLI client: `gcc pbc.c -o pbc -lzmq -Wall`
+
+## Usage
+
+* Get help info: `./pbc -h`
+* **TODO**
```

Comment:
Link the builders/rpc-commands.md  page?

Reply:
Yes, good idea. Will do that